wallet-new/server/Main.hs

```diff
@@ -109,6 +111,7 @@ actionWithLegacyWallet coreConfig walletConfig txpConfig sscParams nodeParams nt
         mconcat [ LegacyPlugins.conversation wArgs
                 , LegacyPlugins.legacyWalletBackend coreConfig txpConfig wArgs ntpStatus
                     [ throttleMiddleware (ccThrottle walletConfig)
+                    , withDefaultHeader (Headers.applicationJson)
```

Comment:
you really like these parens :D

Reply:
Hahaha, yeah, that's called refactoring :joy: And now I don't have hlint to warn me about those since we disabled the extra-parens rule :no_mouth: